### DISCUSSION TITLE
Custom scope's first letter is lowercased in inline label

### DISCUSSION THREAD

**Discussion by instanceofme** [community/community#9790] [Other Feature Feedback, Questions, & Ideas] [answered]
This is nitpicky :)

I created a custom scope named `HAL`, and in the placeholder / inline label in the search, it's called `hAL`:
![Screen Shot 2022-01-07 at 14 49 53](https://user-images.githubusercontent.com/402483/148553425-4fd3f59a-c8eb-4689-a081-5794ed126aba.png)

**Comment by colinwm** [comment, marked as answer]
Woah! Weird bug, will fix that :sweat_smile:

**Reply by colinwm** [reply]
Fixed! Thanks for the report.
